en/components/grid.md

```diff
@@ -805,9 +805,7 @@ Here is a list of all public methods exposed by **igx-grid**:
 |`sort(name: string, direction, ignorecase)`|Sorts a single column.|
 |`sort(expressions: Array)`|Sorts the grid columns based on the provided array of sorting expressions.|
 |`clearSort(name?: string)`|If `name` is provided, clears the sorting state of the corresponding column, otherwise clears the sorting state of all columns.|
-|`pinColumn(columnName: string)`|Pin a column by its name|
```

Comment:
Why were pin and unpin removed?

Reply:
the definition of these methods were duplicated